src/debugger/cli-debugger.c

```diff
@@ -1133,6 +1133,8 @@ void CLIDebuggerAttachBackend(struct CLIDebugger* debugger, struct CLIDebuggerBa
 }
 
 bool CLIDebuggerTabComplete(struct CLIDebugger* debugger, const char* token, bool initial, size_t tokenLen) {
+	(void)initial;
```

Comment:
I had been intentionally leaving this warning in so I could double check to see if there was anything I wanted to do that actually used this warning.

Regardless, it should be `UNUSED(initial);`.